README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# About Cheyenne
+
+![Image of me](https://github.com/cheyennewh/hw01/blob/master/selfie.png)
+
+Hello, my name is **Cheyenne Wakeland-Hart** and I am a rising fourth-year in the college. Most of you don't know me very well, so I'd like to take the opportunity to introduce myself!
+
+## What Do I Study?
+I am a Neuroscience/Psych *double* major within the college and have spent the last two years focusing on memory research. I am interested in the interaction between memory and attentional states and am currently working on projects in:
+* The [Uchicago Memory Lab](https://voices.uchicago.edu/memorylab/)
+* The [CAB Lab](https://cablab.uchicago.edu/research/)
+
+### What Else Do I Care About?
+* **Cat Mom**
+  * I am a cat Mom *est. August last year* to the furball Wonder. He's a very cute, very *loud* orange tabby who loves naps with mom and yelling during Zoom calls.
+![Image of Wonder](https://github.com/cheyennewh/hw01/blob/master/wondercopy.png)
```

Comment:
I'm also partial to cats. Here's Ariel (she's about 9 years old now)

![](https://bensoltoff.com/img/ariel.jpg)